src/vs/workbench/browser/parts/editor/editor.contribution.ts

```diff
@@ -363,9 +363,13 @@ registry.registerWorkbenchAction(SyncActionDescriptor.from(FocusLastGroupAction)
 registry.registerWorkbenchAction(SyncActionDescriptor.from(FocusPreviousGroup), 'View: Focus Previous Editor Group', CATEGORIES.View.value);
 registry.registerWorkbenchAction(SyncActionDescriptor.from(FocusNextGroup), 'View: Focus Next Editor Group', CATEGORIES.View.value);
 registry.registerWorkbenchAction(SyncActionDescriptor.from(FocusLeftGroup, { primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KEY_K, KeyMod.CtrlCmd | KeyCode.LeftArrow) }), 'View: Focus Left Editor Group', CATEGORIES.View.value);
+registry.registerWorkbenchAction(SyncActionDescriptor.create(FocusLeftGroupWithoutWrap, FocusLeftGroupWithoutWrap.ID, undefined), 'View: Focus Left Editor Group Without Wrap', CATEGORIES.View.value);
```

Comment:
As I mentioned before, I do not feel like these commands should appear in the command palette but only be available as commands for someone to assign. I feel this is making the list of commands otherwise a lot more complex without much value.

Reply:
By not supplying a label, they do not appear in the command palette, don't they? How would i register the action otherwise?

Reply:
@Agreon you can use `CommandRegistry`:

e.g. https://github.com/microsoft/vscode/blob/6173cf8a99005aa90ee25b92c8385395caccb8bd/src/vs/workbench/browser/parts/editor/editorCommands.ts#L542-L544